lib/active_record/dynamic_timeout/extensions/postgres_adapter_extension.rb

```diff
@@ -22,6 +28,10 @@ def supports_dynamic_timeouts?
 
     private
 
+    def set_to_default_timeout?(timeout)
+      timeout.nil? || timeout == ":default" || timeout == :default
```

Comment:
The string form of ":default" is kind of odd. Are you sure you need that?

Also, would it make sense to document this `:default` convention, or is that strictly internal?

Reply:
This is copying rails internals for setting the timeout from the config, within the abstract postgres adapter class.

Reply:
See https://github.com/rails/rails/blob/main/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L970-L984

Reply:
Ok!